### ISSUE TITLE
enum string type field in formData with required=true always send the first value.

### ISSUE THREAD

**Issue by SongJoey** [swagger-api/swagger-ui#4693]
### Q&A (please complete the following information)
 - OS: macOS 
 - Browser: Chrome
 - Version: 63
 - Method of installation: docker
 - Swagger-UI version: latest from docker hub: 2376859c51c8
 - Swagger/OpenAPI version: Swagger 2.0

### Content & configuration
formData field with type:string, required: true and enum fields, whatever you choose, it always send the first enum value.

Example Swagger/OpenAPI definition:

add one field to the official pet store example, whatever you choose, it will always send hello.

```
      - in: formData
        name: type
        type: string
        required: true
        enum:
          - hello
          - world
```

### Expected behavior
swagger-ui will send the enum value I choose instead of the first enum value.

### Screenshots
![image](https://user-images.githubusercontent.com/1330008/42266550-4e2dfc38-7fa9-11e8-8ab2-c3c1f713044f.png)

![image](https://user-images.githubusercontent.com/1330008/42266567-5eeb389c-7fa9-11e8-8e7c-27d7aabbfa9a.png)

I've searched in existing issue but didn't find any useful information. thank you!

**Comment by hkosova** [comment]
Interesting... This issue seems to affect all parameter types - path, query, etc., in both 2.0 and 3.0 definitions.

**Comment by SongJoey** [comment]
after some test work I believe it's "required" attribute which caused this issue. when I remove "required" line from definition, it works as expected, except there's a blank option which can cause field missing.

**Comment by shockey** [comment]
Confirmed!

**Comment by bestmike007** [comment]
To reproduce this bug: https://petstore.swagger.io/?url=https://gist.githubusercontent.com/bestmike007/d465fddf34b69f41bbd8533f837a3a73/raw/8196b3624fe486fb727f1fdae485002da375343e/petstore.json#/pet/findPetsByStatus

It goes wrong when an enum field is required and default value is set here: https://github.com/swagger-api/swagger-ui/blob/master/src/core/components/parameter-row.jsx#L43-L57

Line 43 should use `param` rather than `parameter` (latter is the parameter definition with meta) to retrieve the current paramValue.

I'll submit a PR to fix this.

**Comment by bestmike007** [comment]
@SongJoey it's fixed in v3.17.4.
